mir-module/src/main/resources/xsl/editor/xeditor2mods.xsl

```diff
@@ -215,14 +215,19 @@
           <xsl:copy-of select="node()[name()!='mods:namePart']" />
         </xsl:when>
         <xsl:otherwise>
-          <xsl:copy-of select="node()" />
+          <xsl:copy-of select="node()[name()!='mods:role']" />
```

Comment:
It should be sufficient to just switch from `copy-of` to `apply-templates` (for all nodes), i.e.

```
<xsl:choose>
  <xsl:when test="@simpleEditor">
    <xsl:apply-templates select="node()[name()!='mods:namePart']"/>
  </xsl:when>
  <xsl:otherwise>
    <xsl:apply-templates/>
  </xsl:otherwise>
</xsl:choose>
```
The special treatment `<xsl:if test="mods:role/mods:roleTerm">` is then unnecessary. I've tested this both with a simple editor (i.e. `editor-dynamic.xed/`) and a normal editor (i.e. `editor-admins.xed`). I didn't notice any negatice or unintended effects and this should provide more flexibility for future modifications.